packages/credit-card-integration/src/credit-card-payment-strategy.ts

```diff
@@ -20,7 +20,7 @@ export default class CreditCardPaymentStrategy implements PaymentStrategy {
     protected _hostedForm?: HostedForm;
     protected _shouldRenderHostedForm?: boolean;
 
-    constructor(private _paymentIntegrationService: PaymentIntegrationService) {}
+    constructor(protected _paymentIntegrationService: PaymentIntegrationService) {}
```

Comment:
```suggestion
    constructor(protected paymentIntegrationService: PaymentIntegrationService) {}
```
Makes sense to me, so we'd better remove `_` as it is no longer private.